cms/static/js/views/settings/main.js

```diff
@@ -164,7 +209,26 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var value;
+        var index = event.currentTarget.getAttribute('data-index');
         switch (event.currentTarget.id) {
+        case 'course-learning-info-' + index:
```

Comment:
Individual views are not updating. Is it ? 
I just put them all in already used place ( Parent View ) .